### DISCUSSION TITLE
Commenting doesn't refresh the page

### DISCUSSION THREAD

**Discussion by TheCarpetMerchant** [community/community#56214] [Mobile]
If you add a comment or edit one, the page doesn't refresh to show the changes.

**Comment by eliperkins** [comment]
Hey @TheCarpetMerchant , thanks for this feedback! It sounds like a bug to me. Could I get some more info from you to help debug?

- What platform are you using (iOS or Android)?
- What version of GitHub Mobile are you using?

**Comment by TheCarpetMerchant** [comment]
Android 1.113.1

**Comment by TheCarpetMerchant** [comment]
Well damn, it works now. Oh well
